packages/valory/skills/counter/tests/test_counter.py

```diff
@@ -68,6 +68,14 @@ def setup_class(cls, **kwargs: Any) -> None:
             AbciDialogues, cls._skill.skill_context.abci_dialogues
         )
 
+    @classmethod
+    def setup(
+        self,
+    ) -> None:
+        """Setup test."""
+
+        self.abci_counter_handler.tx_count = 0
```

Comment:
Since before we used to load skill for every test separately, the tests did not effect the `tx_count` for other tests. Since now we load the skill once, we need to reset the `tx_count` for every test